pkg/validator/validate.go

```diff
@@ -0,0 +1,34 @@
+package validator
+
+import (
+	"errors"
+	"strings"
+
+	"gopkg.in/go-playground/validator.v9"
```

Comment:
as this has been added, it'd be nice to do this in a specific PR and move all existing code to this.